diff --git a/cmake/configs/default.cmake b/cmake/configs/default.cmake
index b893819..061658d 100644
--- a/cmake/configs/default.cmake
+++ b/cmake/configs/default.cmake
@@ -108,6 +108,7 @@ hunter_config(spdlog VERSION 1.0.0-p0)
 hunter_config(szip VERSION 2.1.0-p1)
 hunter_config(wxWidgets VERSION 3.0.2)
 hunter_config(ZeroMQ VERSION 4.1.4)
+hunter_config(GoogleProtocolBuffers VERSION 2.0.0-beta-2 CMAKE_ARGS "protobuf_BUILD_TESTS=OFF")
 
 if(MINGW OR MSVC80)
   hunter_config(GTest VERSION 1.7.0-hunter-6)
